test/e2e/bootstrapper_test.go

```diff
@@ -26,24 +30,48 @@ func init() {
 	pflag.StringVar(&installDir, "install-dir", "C:\\k", "Installation directory")
 }
 
-// This test will only test whether or not the bootstrapper was able to start the required services. It gives no
-// guarentees that the underlying executables (kubelet) are functioning as they should.
+// TestBootstrapper tests that the bootstrapper was able to start the required services
 // TODO: Consider adding functionality to this test to check if the underlying processes are running properly,
 //  	 otherwise keep that functionality contained within other future tests
 func TestBootstrapper(t *testing.T) {
+	var kubeletRunningOnStart bool
 	// We make a best effort to ensure that there is a ignition file on the node.
 	// TODO: Consider doing the same with kubelet. We can either provide our own or download it from the internet
 	// 		 if we choose to download it, we will have to compare expected vs actual SHA hashes for security reasons
 	ensureIgnitionFileExists(t, ignitionFilePath)
+
+	// If the kubelet is not running yet, we can run disruptive tests
+	if svcExists(t, bootstrapper.KubeletServiceName) {
```

Comment:
shouldn't this be checking for the `!running` state?

Reply:
It should be checking both, i'll change that